core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -253,14 +257,22 @@ private void sendInvitationMessage(UserDTO requestingUser, User buddyUserEntity,
 			String bodyTemplateName = "buddy-invitation-body";
 			String requestingUserName = StringUtils
 					.join(new Object[] { requestingUser.getFirstName(), requestingUser.getLastName() }, " ");
+			String requestingUserMobileNumber = requestingUser.getMobileNumber();
```

Comment:
What about making this number the "from number"? Plivo allows specifying the" from", but I'm not sure whether we can do that per request.